install/0000_30_machine-api-operator_01_images.configmap.yaml

```diff
@@ -23,5 +23,6 @@ data:
       "clusterAPIControllerIBMCloud": "quay.io/openshift/origin-ibmcloud-machine-controllers",
       "clusterAPIControllerOvirt": "quay.io/openshift/origin-ovirt-machine-controllers",
       "clusterAPIControllerPowerVS": "quay.io/openshift/origin-powervs-machine-controllers",
-      "clusterAPIControllerVSphere": "quay.io/openshift/origin-machine-api-operator"
+      "clusterAPIControllerVSphere": "quay.io/openshift/origin-machine-api-operator",
+      "clusterAPIControllerNutanix": "quay.io/openshift/origin-nutanix-machine-controllers"
```

Comment:
Could you add this at the correct place alphabetically please, I think the rest of this list is already alphabetical provider wise

Reply:
Sure